lib/l10n/intl_en.arb

```diff
@@ -23,5 +23,7 @@
   "continueBtn":"Continue",
   "cancelBtn":"Cancel",
   "homePageViewTitle":"Home Page",
-  "linearProgressIndicator":"Linear progress indicator"
+  "linearProgressIndicator":"Linear progress indicator",
+  "facebookSigninErrorMessage": "Unable to sign you in at the moment. Please try again later",
```

Comment:
The message is very generic, u don't need to specify the name as Facebook sign in failed message

Reply:
Will update it in the next PR